schema/show.js

```diff
@@ -131,6 +135,15 @@ const ShowType = new GraphQLObjectType({
       type: Location.type,
       resolve: ({ location, fair_location }) => location || fair_location,
     },
+    city: {
+      type: GraphQLString,
+      resolve: ({ location, partner_city }) => {
+        if (location && isExisty(location.city)) {
+          return existyValue(location.city);
```

Comment:
Just out of curiosity, seeing as `isExisty` is already used in the condition, couldn’t this just have been `return location.city` ?